examples/irsa/README.md

```diff
@@ -28,7 +28,7 @@ Replace `<ACCOUNT ID>` with your AWS account ID in `cluster-autoscaler-chart-val
 Install the chart using the provided values file:
 
 ```
-helm install --name cluster-autoscaler --namespace kube-system stable/cluster-autoscaler --values=cluster-autoscaler-chart-values.yaml
+helm install cluster-autoscaler --namespace kube-system autoscaler/cluster-autoscaler-chart --values=cluster-autoscaler-chart-values.yaml
```

Comment:
> the Helm chart has been moved to the kubernetes/autoscaler repository

Can you provide a reference for this move? Where is the new chart? I can't find it.

Also, it might be best to now show how to add the new helm autoscaler repository to helm if it's not included by default 🙂

Reply:
Sorry @max-rocket-internet , I think I used a wrong link in my initial commit message. The reference to the change is here:
https://github.com/helm/charts/tree/master/stable/cluster-autoscaler#deprecated

My PR fixes the change in the new chart:
https://github.com/kubernetes/autoscaler/blob/736c624181edbc9851d2f866778c782e89de165a/charts/cluster-autoscaler-chart/templates/serviceaccount.yaml#L8

My last commit adds the new helm autoscaler repository.

Reply:
@max-rocket-internet is this LGTY ?

Reply:
for what is worth, I can confirm this change. maybe noteworthy: the moved autoscaler chart has also been updated to only support Helm v3 whereas the old one was Helm v2